ansible/roles/grafana/COPYRIGHT

```diff
@@ -0,0 +1,18 @@
+debops.grafana - Install Grafana on a Debian/Ubuntu host using Ansible
+
+Copyright (C) 2019 Innobyte Bechea Leonardo https://www.innobyte.com/
+Copyright (C) 2019 Innobyte Alin Alexandru https://www.innobyte.com/
```

Comment:
Can you change the copyright attributions to something [like this](https://github.com/debops/debops/blob/master/ansible/roles/tinyproxy/COPYRIGHT#L3-L5) instead? That way the script that builds the manual pages can correctly extract the authorship information from the copyright files.

Reply:
changed, as you requested